samples/nrf9160/http_application_update/src/main.c

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <stdint.h>
+#include <zephyr.h>
+#include <flash.h>
+#include <bsd.h>
+#include <nrf_socket.h>
+#include <download_client.h>
+#include <logging/log.h>
+#include <gpio.h>
+#include <flash_map.h>
```

Comment:
Are `flash_map.h` and `dfu/mcuboot.h` necessary? I can't see where they are used.

Reply:
@lemrey No, they are not needed and are removed now.